### PR TITLE
Added latency measurement for backend datastores

### PR DESCRIPTION
**Pull request by jshaofuturewei**
1. Get latencies for giving address 
2. Added summary to get average latencies

### CODE REVIEW

File: pkg/network/latency/summary.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package latency
+
+import "fmt"
+
+type distribution struct {
+	Average int64
+}
+
+type summary struct {
+	All     *distribution
```
Comment (pdgetrf) [inline pkg/network/latency/summary.go:10]: what does "All" mean here?

File: pkg/network/latency/summary.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	Success *distribution
+}
+
+func makeAverage(x []int64) (out distribution) {
```
Comment (pdgetrf) [inline pkg/network/latency/summary.go:14]: probably the function name average() would do

File: pkg/network/latency/measure.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	Duration time.Duration
+}
+
+type Measurements []Sample
+
+func (m *Measurements) Append(s Sample) {
+	*m = append(*m, s)
+}
+
+func (m *Measurements) InsuccessCount() (out int) {
```
Comment (pdgetrf) [inline pkg/network/latency/measure.go:18]: Insuccess --> Nonsuccess

**Review by pdgetrf — APPROVED** [review]
LGTM

some suggestion about wording

**Comment by jshaofuturewei** [comment]
Updated based on Peng Du's comments

File: pkg/network/latency/summary.go (hunk 1 above)
Comment (jshaofuturewei) [reply pkg/network/latency/summary.go:10]: All means all the connect results(including the successful ones and the failure ones)

**Pull request merged**